src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -0,0 +1,100 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    ''' <summary>
+    ''' Represents a field of a tuple type (such as (int, byte).Item1)
+    ''' that doesn't have a corresponding backing field within the tuple underlying type.
+    ''' Created in response to an error condition.
+    ''' </summary>
+    Friend Class TupleErrorFieldSymbol
```

Comment:
fixed

---
In reply to: [71448381](https://github.com/dotnet/roslyn/pull/12596#discussion_r71448381) [](ancestors = 71448381)